code/modules/projectiles/guns/energy/magic.dm

```diff
@@ -224,6 +243,11 @@ obj/item/weapon/gun/energy/staff/focus/attack_self(mob/living/user as mob)
 	max_shots = 5
 	projectile_type = /obj/item/projectile/magic/fireball
 
+/obj/item/weapon/gun/energy/wand/fire/pickup(mob/living/user as mob)
+	if(!user.is_wizard())
+		user << "<span class='warning'>When you pick up \the [src], you can feel alarming pulses of heat rushing up your arms.  You probably shouldn't try to use this...</span>"
+		user.make_dizzy(120)

```

Comment:
Probably a better idea to do burn or something instead.

Reply:
These dizziness things are not the punishment for picking them up.  It is to make your screen shake for a moment to warn you that this thing is dangerous.  It's using the items that invokes the major punishment.

Punishment stacking is not necessary for just picking up a wizard item.